src/app/state/rest/index.ts

```diff
@@ -305,7 +305,7 @@ export const useCorrespondence = (options?: Options) => {
   })
 }
 
-export const useCompleteCase = (options?: Options) => {
+export const useCompleteCase = (id: Components.Schemas.Case["id"], options?: Options) => {
```

Comment:
"id" is not used?

Reply:
Will make `id` optional and set `lazy` option